onboarding/managers.md

```diff
@@ -7,43 +7,33 @@ description: Things for managers to remember during the early days of onboarding
 
 ### Before Day 1
 
-- [ ] Choose a peer mentor for the new hire
-- [ ] Choose a spirit guide for the new hire (consult with People Ops)
-- [ ] Create
-      [an onboarding github issue](https://github.com/artsy/potential/issues/new?template=engineering-onboarding.md)🔒
-      based on the onboarding template. Also invite the team to say hi, as this is a place that the new hire should
-      always be able to find easily in the future.
-- [ ] Set first-week schedule and let the new hire know their first day schedule on their private email account, as
-      there will be a period where their machine is still being setup and they won’t yet have access to their
-      calendar.
+- [ ] Choose an [onboarding mentor](/onboarding/mentors.md#the-onboarding-mentor) for the new hire. 
+- [ ] Create [an onboarding github issue](https://github.com/artsy/potential/issues/new?template=engineering-onboarding.md) based on the onboarding template. Tag the onboarding mentor and label the issue as `onboarding`. Also invite the team to say hi, as this is a place that the new hire should always be able to find easily in the future.
+- [ ] Set first-week schedule
 - [ ] Invite new hire to all engineering-wide meetings
-- [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md) and update the [onboarding calendar](https://calendar.google.com/calendar/b/1?cid=YXJ0c3ltYWlsLmNvbV9jc2RocTFnNHNyMmhhY2Nvc3RmMTA1bTEwc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)🔒 accordingly
+- [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md) and update the [onboarding calendar](https://www.notion.so/artsy/Onboarding-Rotations-073f98b82d4840aaa0b2bba71343e7cc)🔒 accordingly
 
 ### Day 1
 
-- [ ] Organize lunch outing (probably to Tataki!)
```

Comment:
tataki forever!

Reply:
ahh now I get this reference 😆 

https://github.com/artsy/radiation/blob/88050cf405093af7580c5f984d1e83a4c89f5616/spec/services/review_service_spec.rb#L216